howto/addons/migrate.md

```diff
@@ -1,9 +1,8 @@
-Starting with Anbox Cloud 1.12, the following hooks are deprecated and should be replaced:
+Starting with Anbox Cloud 1.12, use the following new hooks instead of the deprecated ones:
 
-- `install` should be replaced with `pre-start`
-- `prepare` should be replaced with `post-start`
-- `restore` should be replaced with `pre-start`
-- `backup` should be replaced with `post-stop`
+* Use `pre-start` instead of `install` and `restore`
```

Comment:
not native speaker but I feel the following feel more nature 
`install` and `restore` 
-> 
`install` or `restore`